docs/apm/correlations.asciidoc

```diff
@@ -2,17 +2,15 @@
 [[correlations]]
 === Find latency and error correlations
 
-**Correlations** surface attributes of your data that are potentially correlated with high-latency or erroneous transactions.
-By default, a number of attributes commonly known to cause performance issues, like version,
-infrastructure, and location, are included, but all are completely customizable to your APM data.
-Find something interesting? A quick click of a button will auto-query your data as you work to resolve the underlying issue.
-
-For example, a site reliability engineer, who is responsible for keeping production systems up and running,
-notices an increase in latency in certain transactions.
-Analyzing metadata or tags that exist in high-latency transactions but not in lower-latency transactions
-can potentially point towards the root cause.
-They may find that a particular piece of hardware, like a host or pod, has failed, increasing latency.
-Or, perhaps set of users, based on IP address or region, is facing increased latency due to local data center issues.
+**Correlations** surface attributes of your data that are potentially correlated
+with high-latency or erroneous transactions. For example, a site reliability
+engineer, who is responsible for keeping production systems up and running,
+notices an increase in latency in certain transactions. Analyzing attributes
+that exist in high-latency transactions but not in lower-latency transactions
```

Comment:
It's not quite this black and white (exist/not exist)...it's more the skew of the latency distribution the attributes cause. E.g.

`Identifying attributes that are responsible for higher latency transactions can potentially point towards the root cause.`